idmega2000/visitrail
# #166255694  Re-organise the root folder structure

**What does this PR do?**

It restructures the root folder and put the manage.py file in the root folder

**Description of Task to be completed?**

- restructure the root folder

**How should this be manually tested?**
- Fetch the branch `git fetch origin ch-re-organise-root-folder-structure-166255694`
- Check out into the branch `ch-re-organise-root-folder-structure-166255694`
- The manage.py file should be in the root folder

**Any background context you want to provide?**

N/A

**What are the relevant pivotal tracker stories?**

[#166255694](https://www.pivotaltracker.com/story/show/166255694)

**Screenshots (if appropriate)**

N/A
